crates/shared/src/price_estimation/competition/mod.rs

```diff
@@ -121,6 +121,14 @@ impl<T: Send + Sync + 'static> CompetitionEstimator<T> {
                     elapsed = ?start.elapsed(),
                     "new price estimate"
                 );
+                if let Err(ref e) = result {
+                    tracing::warn!(?query,
+                        ?e,
+                        estimator = name,
+                        requests = requests.len(),
+                        elapsed = ?start.elapsed(),
+                        "failed to get price estimate");
+                }
```

Comment:
Don't we already basically have this log by using this query in Kibana?

https://production-6de61f.kb.eu-central-1.aws.cloud.es.io/app/r/s/2YsfE

Reply:
umm you are right, for whatever reason I thought we don't print `debug` and I didn't want to be too spammy in the logs. That will do! thank you!